alethzero/DappLoader.cpp

```diff
@@ -96,13 +97,31 @@ DappLocation DappLoader::resolveAppUri(QString const& _uri)
 
 void DappLoader::downloadComplete(QNetworkReply* _reply)
 {
+	QUrl requestUrl = _reply->request().url();
+	if (m_pageUrls.count(requestUrl) != 0)
+	{
+		//inject web3 js
+		QByteArray content = "<script>\n";
+		content.append(web3Content());
+		content.append("</script>\n");
+		content.append(_reply->readAll());
```

Comment:
It works in chromium and we don't have to care about compatibility with anything else here